package.json

```diff
@@ -414,5 +455,7 @@
     "vscode-languageclient": "3.5.0",
     "vscode-languageserver-protocol": "3.5.0",
     "vscode-languageserver-types": "3.5.0"
-  }
+  },
+  "requiredDotNetCoreSDK": "2.1.0",
+	"projectTemplateNupkg": "resources/Microsoft.DataTools.PostgreSql.ProjectTemplate.1.0.0.nupkg"
```

Comment:
This will be displayed at 3 places:

1. In the file explorer you can right click on a pgproj file.
 2. If a pgproj file is open in editor, you can right click anywhere on the editor and it will show the command.
3. If pgproj is open in editor, you will see it in command palette.